README.md

```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
 Keyboard innput can be given at run time<br/> 
 robot will update path taking obstacle avoidance in account as goal is changed by keys i , l j to move it forword backword rigth left 
 
+# How to launch
+### System requirement
+Ubuntu 18/20<br/>
+ROS Melodic/Neotic (Ros1)<br/>
+Rviz (present by default if not [refer](http://wiki.ros.org/rviz/UserGuide))<br/>
+For further ROS package dependencies [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
+#### Repository requirement
+For running the simulation you must have Maxl Planner install which is mention above<br/>
+For installing Maxl planner [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
```

Comment:
NOTE: This is the wrong link. This google doc is only to be given to an employee not the customer. Customer have to go through read the docs to get their information. Please remove this link ASAP and update the link with read the docs link. Also, please be more descriptive in readme.